test/message/test_runner_describe_it.js

```diff
@@ -339,3 +328,47 @@ describe('timeouts', () => {
     setTimeout(done, 10);
   });
 });
+
+describe('successful thenable', () => {
+  it('successful thenable', () => {
+    let thenCalled = false;
+    return {
+      get then() {
```

Comment:
Honestly personally I think anything related to non-native promises isn't important to even test or support anymore probably :D?

Reply:
See https://github.com/nodejs/node/pull/43998?notification_referrer_id=NT_kwDOANpZjbM0MDcxMTMyMzExOjE0MzA5Nzcz#discussion_r929874573.